source/uwp/Renderer/lib/XamlHelpers.cpp

```diff
@@ -745,6 +745,11 @@ namespace AdaptiveCards::Rendering::Uwp::XamlHelpers
         if (label)
         {
             winrt::AutomationProperties::SetLabeledBy(actualUIElement, label);
+
+            if (!adaptiveInput.Label().empty())
```

Comment:
I wonder if we need to move this outside the parent if statement

Reply:
Just updated :)